packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -263,69 +425,61 @@ function joinMeeting(meeting) {
   });
 }
 
-// In order to simplify the state management needed to keep track of our button
-// handlers, we'll rely on the current meeting global object and only hook up event
-// handlers once.
+// The function is called when selecting the auth type in the form
+// and presenting the correct form associated with the selected auth type
+function handleAuthTypeChange() {
```

Comment:
This function is in the middle of meeting action functions. Please move it to the auth section.